libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetworkTests/MockNetworkTests.m

```diff
@@ -57,6 +57,10 @@ - (void)testNetworkBasics {
     CSFSalesforceAction *action1 = [[CSFSalesforceAction alloc] initWithResponseBlock:nil];
     CSFSalesforceAction *action2 = [[CSFSalesforceAction alloc] initWithResponseBlock:nil];
     
+    // Setting verbs and methods (values do not matter - but can't be nil otherwise it confuses duplicateActionInFlight/isEqualToAction
+    action1.method = action2.method = @"POST";
```

Comment:
That's the fix for the other test - basically isEqualToAction (called by duplicateActionInFlight) doesn't like nil verbs/methods. NB Real life actions would always have a verb and method.